### PR TITLE
Add default Value to the picker

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

File: the-10-min/src/Picker/GestureHandler.tsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ const GestureHandler = ({ value, max, defaultValue }: GestureHandlerProps) => {
     velocity: velocity.y,
     state,
     snapPoints,
+    offset: new Value(-ITEM_HEIGHT * 34),
```
Comment (efrister) [inline the-10-min/src/Picker/GestureHandler.tsx:37]: @wcandillon Sorry if I'm missing this, but why the number "34"? What does this represent?

Comment (eqlion) [reply the-10-min/src/Picker/GestureHandler.tsx:37]: I believe that's a hardcoded default value

**Comment by richardantao** [comment]
You linked this commit to a youtube comment that asked how to access the current value of the Picker. Looking at the code, it's not immediately evident to do that. Use snappoints to match the index of the value maybe? Do you mind elaborating on how to access the value?

File: the-10-min/src/Picker/GestureHandler.tsx (hunk 1 above)
Comment (Mario857) [reply the-10-min/src/Picker/GestureHandler.tsx:37]: I do it like this:
 
 GestureHandler.tsx
 ```
 offset: new Value(0),
 ```
 
 AnimationHelpers.tsx
 ```
const defaultIndex = findIndex(values, (v) => v.value === defaultValue);

cond(not(init), [
    set(offset, -ITEM_HEIGHT * defaultIndex),
    set(state.position, offset),
    set(init, 1),
  ]),
 ```

File: the-10-min/src/Picker/GestureHandler.tsx [hunk 2]
```diff
@@ -33,6 +34,7 @@ const GestureHandler = ({ value, max, defaultValue }: GestureHandlerProps) => {
...
   });
   useCode(() => [set(value, add(translateY, ITEM_HEIGHT * 2))], []);
```
Comment (Mario857) [inline the-10-min/src/Picker/GestureHandler.tsx:39]: And for picker values I use this trick, onValueChange is prop

```
  useCode(() => [set(value, add(translateY, ITEM_HEIGHT * 2))], []);
  
  useCode(() => {
    return call([translateY], ([currentValue]) => {
      const selectedIndex = Math.round(-currentValue / ITEM_HEIGHT);
      const newValue = values[selectedIndex]?.value;

      if (onValueChange && newValue) {
        onValueChange(newValue, selectedIndex);
      }
    });
  }, [translateY]);
```

Comment (Mario857) [reply the-10-min/src/Picker/GestureHandler.tsx:39]: Also keep in mind that you can't call setState in onValueChange callback It doesn't work, I use refs.
  const currentValueRef = React.useRef(defaultValue);
  
 <GestureHandler
                values={values}
                max={values.length}
                value={translateY}
                onValueChange={(value: any) => {
                  currentValueRef.current = value;
                }}
                defaultValue={defaultValue}
              />

**Pull request merged**